aws_lambda_builders/workflows/java_maven/maven_validator.py

```diff
@@ -0,0 +1,64 @@
+"""
+Maven Binary Validation
```

Comment:
Is this radically different from the gradle validator? If so, then thats fine. Im  just trying to see if they work in very similar ways.

Reply:
It works similarly to Gradle validator. It basically runs `mvn -version` and checks the printed out jvm version.